### ISSUE TITLE
update conda channel failing on jobwatch package

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3#1149]
**Description**

The action to update the masters conda channel with a recent jobwatch masters package looks to have failed

https://github.com/sot/jobwatch/actions/runs/5808795842/job/15752626512

I've tried re-running the actions.  Not sure what else to troubleshoot here if the local runner can't find conda?

**Comment by javierggt** [comment]
Ah! There was another issue, and that is that the conda setup, as shown in the twiki, was to install it in `/export/jgonzale/github-workflows/miniconda3-shiny`, but it should be `/export/jgonzale/github-workflows/miniconda3`. I created a symlink so the runners can find it, but this should be fixed next time the runners are updated and the symlink removed.

**Comment by javierggt** [comment]
Note: I did not update the runners, I just updated the conda environment which is included in the PATH of the runners.

**Comment by jeanconn** [comment]
Right, though basically that is all the code that is run by the self-hosted runners just not their configurations on github.
